src/Sylius/Bundle/CoreBundle/Controller/CheckoutController.php

```diff
@@ -193,6 +193,20 @@ public function finalizeAction(Request $request, OrderInterface $order)
     }
 
     /**
+     * @return Response
+     */
+    public function thankYouAction()
+    {
+        $id = $this->get('session')->get('sylius_order_id');
+        $order = $this->findOrderOr404($id);
+        $customerName = $order->getCustomer()->getFullName();
+        
+        return $this->render('SyliusWebBundle:Frontend/Checkout/Step:thankYou.html.twig', array('customer_name' => $customerName));
```

Comment:
We should pass whole order object here and get the customer full name in template.